arangod/Aql/CollectionAccessingNode.cpp

```diff
@@ -83,3 +84,12 @@ void CollectionAccessingNode::toVelocyPack(arangodb::velocypack::Builder& builde
     builder.add("restrictedTo", VPackValue(_restrictedTo));
   }
 }
+
+void CollectionAccessingNode::toVelocyPackHelperPrimaryIndex(arangodb::velocypack::Builder& builder) const {
+  auto col = _collection->getCollection();
+  builder.add(VPackValue("indexes"));
+  col->getIndexesVPack(builder, false, false, [](arangodb::Index const* idx) {
+      return (idx->type() == arangodb::Index::TRI_IDX_TYPE_PRIMARY_INDEX ||
+              idx->type() == arangodb::Index::TRI_IDX_TYPE_EDGE_INDEX);
```

Comment:
it's very confusing that this method is supposed to return the VPack for the primary index, but will apparently also do so for the edge index. This makes no sense.